tests/integration/template_min_case_e3sm_diags_tc_analysis.cfg

```diff
@@ -1,20 +1,20 @@
 [default]
-case = "#expand case_name#"
+case = "#expand case_name_v2#"
```

Comment:
Any test using `tc_analysis` now uses the v2 input data because of https://github.com/E3SM-Project/e3sm_diags/pull/866#issuecomment-2405873119